pkg/model/components/kubescheduler_test.go

```diff
@@ -25,12 +25,14 @@ import (
 )
 
 func buildSchedulerConfigMapCluster() *api.Cluster {
+	usePolicyConfigMap := true
+
 	return &api.Cluster{
 		Spec: api.ClusterSpec{
 			CloudProvider:     "aws",
 			KubernetesVersion: "v1.4.0",
 			KubeScheduler: &api.KubeSchedulerConfig{
-				PolicyConfigMap: "scheduler-config",
```

Comment:
Any reason for removing `PolicyConfigMap`? It seems the name of the ConfigMap changes from `scheduler-config` -> `scheduler-policy` between the 2 versions.

Reply:
Sorry if I'm missing some context, reading through the other PR now :)

Reply:
Yeah, so aftering reading through the PRs, my only question is around why we chose to change the ConfigMap name here, it seems it can break backwards compatability or leave dangling resources. Does scheduler create the ConfigMap `scheduler-policy` by default if it doesn't exist?

Reply:
That's an oversight on my part. I'll rebase and change the name back. Thanks for the catch

Reply:
Actually I just realized. The name "scheduler-config" is not actually in use anywhere except as test data. The actual config name in the cluster is configurable by this PolicyConfigMap field (which has no default value).

Reply:
ahh, missed that it's just test data, thanks!